file.css

```diff
@@ -0,0 +1 @@
+h1 {color:aqua}
```

Comment:
- [ ] It is required that the class name given to the header should be used to add styling to the content of the header. It is also a good practice to use class-selector than element-selector to avoid conflict.

### Project Requirement
![style](https://github.com/MoliBravo/Hello-Microverse/assets/26597432/6da434d0-6acf-4574-88bf-71f2e7310a6d)